mw-webapp/src/styles/_variables.scss

```diff
@@ -31,6 +31,7 @@ $paddingBig: 20px;
 $paddingMedium: 10px;
 $paddingSmall: 5px;
 $gapBig: 20px;
+$gapLarge: 36px;
```

Comment:
@Ekaterina1994 do we need more gaps?

Reply:
I don't now))  It depends on design)

Reply:
removed this variable from global variables.scss

Reply:
so with no such variable styles on differrent pages will not be consistent. If we really need it - better to have it in the global variables

Reply:
Yes, i think its realy need

Reply:
@tritonJS826  You asked me do we need more gaps. So I don't know if in figma we have more different gaps on different pages and blocks then we will shoould add new gaps, paddings and margin. About consistance we could speak when design will be finished. Everything is changes each day.